steno/src/view/components/top_bar.cpp

```diff
@@ -1,20 +1,26 @@
 #include "top_bar.h"
+#include "model/config/config.h"
+#include "model/transcription/transcription_config.h"
 #include "ui_top_bar.h"
 
 #include "colors.h"
 
 #include <QDesktopServices>
-#include <QSignalBlocker>
+#include <QNetworkReply>
 #include <QStyle>
 #include <QUrl>
 
 namespace View
 {
-TopBar::TopBar(std::shared_ptr<Model::IStream> stream, std::shared_ptr<Model::Media> media, QWidget* parent)
+TopBar::TopBar(std::shared_ptr<Model::IStream> stream, std::shared_ptr<Model::Media> media,
+               std::shared_ptr<Model::Transcription> transcription, std::shared_ptr<Model::Config> config,
+               QWidget* parent)
     : QWidget(parent)
     , m_ui(new Ui::TopBar)
     , m_stream(stream)
     , m_media(media)
+    , m_transcription(transcription)
+    , m_config(config)
```

Comment:
Maybe we could store only the transcription config by calling `m_config->transcriptionConfig()` here

Reply:
done